cmd/dmverity-vhd/main.go

```diff
@@ -98,7 +101,32 @@ func fetchImageLayers(ctx *cli.Context) (layers []v1.Layer, err error) {
 
 	// by default, using remote as source
 	var img v1.Image
-	if tarballPath != "" {
+	if tarballPath != "" || dockerDaemon {
+		if dockerDaemon {
+			ctx := context.Background()
+			cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+			if err != nil {
+				return nil, fmt.Errorf("failed to instanciate docker client: %w", err)
```

Comment:
instantiate*

Reply:
Ack, although I think we can close this PR